src/fragments/lib/auth/common/totp/flows.mdx

```diff
@@ -0,0 +1,89 @@
+
+## Setup
+
+<Callout warning>
+
+The following options are only available when starting a new project (via `amplify add auth`). You will not have access to these settings after creation (via `amplify update`).
+
+- Required MFA
```

Comment:
Maybe clarify that this is `Setting MFA to "ON"`

Reply:
Done.